### PR TITLE
bump k8s to 1.6.6

### PR DESCRIPTION
**Pull request by rjtsdl**
**What this PR does / why we need it**:
Bump k8s version to latet 1.6.6

**Special notes for your reviewer**:

**Release note**:

```release-note
Release k8s 1.6.6
```

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/796)

### CODE REVIEW

File: pkg/api/convertertoapi.go [hunk 1]
```diff
@@ -385,7 +385,7 @@ func convertV20160330OrchestratorProfile(v20160330 *v20160330.OrchestratorProfil
 func convertV20170131OrchestratorProfile(v20170131 *v20170131.OrchestratorProfile, api *OrchestratorProfile) {
 	api.OrchestratorType = OrchestratorType(v20170131.OrchestratorType)
 	if api.OrchestratorType == Kubernetes {
-		api.OrchestratorVersion = Kubernetes162
+		api.OrchestratorVersion = KubernetesLatest
```
Comment (rjtsdl) [inline pkg/api/convertertoapi.go]: This should be Latest IMO.

File: pkg/acsengine/const.go [hunk 2]
```diff
@@ -51,6 +51,18 @@ const (
...
+		"hyperkube":    "hyperkube-amd64:v1.6.6",
+		"dashboard":    "kubernetes-dashboard-amd64:v1.6.0",
+		"exechealthz":  "exechealthz-amd64:1.2",
+		"addonresizer": "addon-resizer:1.6",
+		"heapster":     "heapster:v1.2.0",
+		"dns":          "kubedns-amd64:1.7",
+		"addonmanager": "kube-addon-manager-amd64:v6.2",
+		"dnsmasq":      "kube-dnsmasq-amd64:1.3",
+		"pause":        "pause-amd64:3.0",
+		"windowszip":   "v1.6.2intwinnat.zip",
```
Comment (JiangtianLi) [inline pkg/acsengine/const.go]: 1.6.6

File: pkg/acsengine/const.go [hunk 3]
```diff
@@ -61,7 +73,7 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"addonmanager": "kube-addon-manager-amd64:v6.2",
 		"dnsmasq":      "kube-dnsmasq-amd64:1.3",
 		"pause":        "pause-amd64:3.0",
-		"windowszip":   "v1.6.2intwinnat.zip",
+		"windowszip":   "v1.6.6intwinnat.zip",
```
Comment (JiangtianLi) [inline pkg/acsengine/const.go]: 1.6.2 here?

File: pkg/acsengine/const.go (hunk 2 above)
Comment (rjtsdl) [inline pkg/acsengine/const.go]: I will let Jiangtian to confirm here.

File: pkg/acsengine/const.go [hunk 4]
```diff
@@ -51,6 +51,18 @@ const (
 )
 
 var KubeImages = map[api.OrchestratorVersion]map[string]string{
+	api.Kubernetes166: {
+		"hyperkube":    "hyperkube-amd64:v1.6.6",
```
Comment (rjtsdl) [inline pkg/acsengine/const.go]: I only changed hyperkube's version. The others stay the same with 1.6.2

File: pkg/acsengine/const.go (hunk 2 above)
Comment (JiangtianLi) [inline pkg/acsengine/const.go]: yes, we will use v1.6.*intwinnat.zip (with winnat in the name).

**Pull request merged**